assets/app/scripts/extensions/javaLink.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular
-  .module('openshiftConsoleExtensions')
+  .module('openshiftConsoleExtensions', ['openshiftConsole'])
```

Comment:
If other extensions use `openshiftConsoleExtensions`, so we declare it elsewhere?